src/fuzz_introspector/analysis.py

```diff
@@ -82,9 +83,15 @@ def analysis_func(
                            be shown at the top of the report page.
         :type conclusions: List[html_helpers.HTMLConclusion]
 
+        :param json_report: A bool value to declare if the ouput is json string for
+                            the json report (True) or html string for html
```

Comment:
Hmm, am not sure if I like that it's either `json` or `html` because it means in order to collect both we need to run the analysis twice. Could we not make it such that the `.json` content is build up, and also potentially the `html` string which are then accessed through a property on the Python class?

Reply:
Done